readme.md

```diff
@@ -1,4 +1,6 @@
-![Medusa](themes-default/slim/static/images/medusa-logo.png)
+<p align="center">
+  <img src="https://cdn.rawgit.com/pymedusa/medusa.github.io/4360d494/images/logo/new-logo.png"/>
+</p>
 
 [![Build Status](https://travis-ci.org/pymedusa/Medusa.svg?branch=develop)](https://travis-ci.org/pymedusa/Medusa) [![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/pymedusa/Medusa.svg)](http://isitmaintained.com/project/pymedusa/Medusa "Average time to resolve an issue") [![Percentage of issues still open](http://isitmaintained.com/badge/open/pymedusa/Medusa.svg)](http://isitmaintained.com/project/pymedusa/Medusa "Percentage of issues still open") [![codebeat badge](https://codebeat.co/badges/4b801428-c8b3-47aa-83aa-1d82677c52c0)](https://codebeat.co/projects/github-com-pymedusa-medusa-develop) [![XO code style](https://img.shields.io/badge/code_style-XO-5ed9c7.svg)](https://github.com/sindresorhus/xo) [![codecov](https://codecov.io/gh/pymedusa/Medusa/branch/develop/graph/badge.svg)](https://codecov.io/gh/pymedusa/Medusa)
```

Comment:
On the image from Slack, those were centered as well?
![screen shot 2018-04-28 at 9 38 24 pm](https://user-images.githubusercontent.com/10238474/39424592-0e45604a-4c80-11e8-9809-2e80d869d3e8.png)

Reply:
the exclusive features. Is that compared to sonarr and sickrage? Or just to sickrage?